tox.ini

```diff
@@ -0,0 +1,41 @@
+[tox]
+envlist = py37
+skipsdist = True
+[testenv]
+setenv =
+    ENV = dev
+passenv =
+    ENV
+    TAP_SNOWFLAKE_USER
+    TAP_SNOWFLAKE_ACCOUNT
+    TAP_SNOWFLAKE_PASSWORD
+    TAP_SNOWFLAKE_WAREHOUSE
+    TAP_SNOWFLAKE_DBNAME
```

Comment:
Does a test DB actually exist somewhere? If so, maybe we should update the README on where that DB is and how it is being maintained.